src/posts/datatree/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 'Xarray x NASA: New xarray.DataTree for hierarchical data structures'
+date: '2024-12-04'
+authors:
+  - name: Tom Nicholas
+    github: tomnicholas
+  - name: Owen Littlejohns
+    github: owenlittlejohns
+  - name: Matt Savoie
+    github: flamingbear
+  - name: Eni Awowale
+    github: eni-awowale
+  - name: Alfonso Ladino
+    github: aladinor
+  - name: Justus Magin
+    github: keewis
+  - name: Stephan Hoyer
+    github: shoyer
+summary: 'The new xarray.DataTree class allows working with netCDF/Zarr groups, brought to you in collaboration with NASA!'
+---
+
+## TL;DR
+
+`xarray.DataTree` has been released in [v2024.10.0](https://github.com/pydata/xarray/releases/tag/v2024.10.0), and the prototype [`xarray-contrib/datatree` repository](https://github.com/xarray-contrib/datatree) archived, after collaboration between the xarray team and [NASA ESDIS](https://www.earthdata.nasa.gov/about/esdis).
```

Comment:
Is it [ESDIS](https://www.earthdata.nasa.gov/about/esdis), [ESDS](https://earthdata.nasa.gov/about), or [EOSDIS](https://www.earthdata.nasa.gov/about/esdis/eosdis)??? 😅

Reply:
I think "ESDIS" is the way to go, or more specifically, the "ESDIS project". EOSDIS is the software stack for ESDIS, and it's a term that seems to be in the process of being phased out anyway.

Reply:
Is ESDIS the DAAC? I want to refer to whichever the correct name is for the org you represent (and I understand just "NASA" would be far too broad to be accurate).

Reply:
Wait and @eni-awowale and @flamingbear should your affiliations with GES DISC and NSIDC be included explicitly too?

Reply:
Updated to be consistent in https://github.com/xarray-contrib/xarray.dev/pull/736/commits/80a10d209158db9aa81ab66fb1127b31a0e4e252